modules/commerce_bluesnap_currency/src/EventSubscriber/ExchangeRateBluesnap.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\commerce_bluesnap_currency\EventSubscriber;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\commerce_currency_resolver\ExchangeRateEventSubscriberBase;
+use Drupal\Component\Serialization\Json;
+
+/**
+ * Class ExchangeRateBluesnap.
+ *
+ * Fetches all currency conversion rates from bluesnap currency rate API.
+ */
+class ExchangeRateBluesnap extends ExchangeRateEventSubscriberBase {
+
+  /**
+   * Creates a new ExchangeRateBluesnap object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory) {
+    $this->configFactory = $config_factory->get('commerce_currency_resolver.currency_conversion');
```

Comment:
**CHANGE REQUEST**

You need to define the property. Also, you're not storing the factory, you're storing the actual config object i.e.

`$this->config = $config_factory->get('commerce_currency_resolver.currency_conversion');`

Reply:
Done